lib/datadog/tracing/contrib/span_attribute_schema.rb

```diff
@@ -22,6 +22,75 @@ def default_span_attribute_schema?
           Datadog.configuration.tracing.span_attribute_schema ==
             Tracing::Configuration::Ext::SpanAttributeSchema::DEFAULT_VERSION
         end
+
+        # implement this function in all target spans/integrations with spankind
+        def set_peer_service(span)
+          should_set_peer_service(span) && set_peer_service_from_source(span)
+          # if above
+          # then remap + remapped from (SKIP)
+          # else
+          # debug that peer service could not be set
+          # end
+        end
+
+        def should_set_peer_service(span)
```

Comment:
In the spirit of Ruby patterns, a `?` is normally how we define boolean methods that answer a simple question:
```suggestion
        def set_peer_service?(span)
```